atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/impl/ExpectationsMetricsReportingTest.java

```diff
@@ -77,7 +84,18 @@ public void mostKvsBytesReadInSingleCallReportedOnReadInfoWithPresentMaximumByte
                 .containsOnly(bytes);
     }
 
+    @Test
+    public void commitLocksRequestsReported() {
+        SnapshotTransaction.reportExpectationsCollectedData(
+                0L,
+                BLANK_READ_INFO,
+                BLANK_COMMIT_LOCK_INFO.withCellCommitLocksRequested(1).withRowCommitLocksRequested(2),
+                metrics);
+        assertThat(metrics.cellCommitLocksRequested().getSnapshot().getValues()).containsOnly(1);
+        assertThat(metrics.rowCommitLocksRequested().getSnapshot().getValues()).containsOnly(2);
+    }
+
     private void reportMetrics(TransactionReadInfo readInfo) {
-        SnapshotTransaction.reportExpectationsCollectedData(0L, readInfo, metrics);
+        SnapshotTransaction.reportExpectationsCollectedData(0L, readInfo, BLANK_COMMIT_LOCK_INFO, metrics);
```

Comment:
Let's rename reportMetrics to reportMetricsWithReadInfo, since now there's multiple things